premake5.lua

```diff
@@ -137,8 +137,6 @@ filter({"platforms:Linux", "toolset:gcc"})
 
 filter({"platforms:Linux", "language:C++", "toolset:clang"})
   links({
-    "c++",
```

Comment:
we can remove the complete links() command here to keep it clean, the main premake is a mess already